extension/source/background/networks.ts

```diff
@@ -99,6 +99,16 @@ export const builtinProviderConfigs = ensureType<
     chainCurrency: 'ETH',
     networkKey: 'local',
   },
+  mainnet: {
+    blockExplorerUrl: 'http://etherscan.io/',
+    chainId: '0x1', // 1
```

Comment:
Added for fetching method IDs. Parity's mainnet contract has almost all major methods registered.